### ISSUE TITLE
More Color Palettes for Offline Page

### ISSUE THREAD

**Issue by maxcell** [forem/forem#3448]
**Is your feature request related to a problem? Please describe.**
It isn't that there is a problem with it -- it's an opportunity for us to grow in the community could be to allow for more color palette options that offer enough contrast for people with color vision deficiencies.

**Describe the solution you'd like**
Having few options potentially on the page that would be selected through a dropdown. We could implement static palettes that work for people and they can switch between them. That way everyone can enjoy it.

**Describe alternatives you've considered**
The alternative is keeping it the way that it is. Someone might even try to suggest that as a user you should be able to choose them all yourself. Which could be investing a bit too much to an experience but I think this is one of the few places in the site that uses a lot of color that it could actually have impact.

**Additional context**
Here are screenshots of the various ways you could experience this page with color vision deficiencies using the macOS app [Sim Daltonism](https://michelf.ca/projects/sim-daltonism/).

Current Color Palette in Common Vision
![offline screenshot for current color selection](https://user-images.githubusercontent.com/8431042/61049466-91f88400-a3b2-11e9-81b8-94b82819f5cb.png)

Current Color Palette in Deuternopia (*green-blind* in red-green)
![offline screenshot for Deuternopia](https://user-images.githubusercontent.com/8431042/61048984-590bdf80-a3b1-11e9-888f-7fb8d0cb627c.png)

Current Color Palette in Protanopia (*red-blind* in red-green and also impacts blue-yellow)
![offline screenshot for ](https://user-images.githubusercontent.com/8431042/61049147-bc960d00-a3b1-11e9-94d8-00e5bd8c800c.png)

Current Color Palette in Tritanopia (hard to distinguish blue light found in blue-green)
![offline screenshot for Tritanopia](https://user-images.githubusercontent.com/8431042/61049301-10085b00-a3b2-11e9-9a13-29ce74417eb7.png)

**Comment by aspittel** [comment]
Oh this is an awesome idea! Thank you so much for recommending it! Would love to see this implemented or for anyone in the community to take it on. Thanks again!

**Comment by maxcell** [comment]
I don't mind taking a shot at it!

**Comment by tworrell** [comment]
I'm going to try my hand at this!

**Comment by hydrosquall** [comment]
@tworrell and I paired on this at the Hacktoberfest kickoff this evening, there will be a PR up for this soon! For reference, we are working on the self-contained file in https://github.com/thepracticaldev/dev.to/blob/master/public/offline.html . 

Based on discussing with @jessleenyc and @benhalpern, technical constraints for this page are to not use a framework (like Preact) and to avoid using external javascript dependencies, so this is a pure (ES6 based on existing usage of const/arrow functions) Javascript solution.

While working this issue, we noticed that the code for coloring the DEV.TO stripes is not synchronized with the colors used in the drawing picker. We are going to tackle the first part first (color accessibility for the drawing tool), and approach suggest tackling redrawing the stripes in a future PR.
